exercises/D-logical-operators/exercise.js

```diff
@@ -11,29 +11,56 @@ var cssLevel = 4;
 
 // Finish the statement to check whether HTML, CSS knowledge are above 5
 // (hint: use the comparison operator from before)
-var htmlLevelAbove5;
```

Comment:
Here you could have `htmlLevelAbove5 = htmlLevel > 5` and this would behave the same as the `htmlLevelAbove5` function. This applies to all the other variables in here, so `cssLevelAbove5`, `cssAndHtmlAbove5` and `cssOrHtmlAbove5`. So for example you could also have `cssAndHtmlAbove5 = htmlLevel > 5 && cssLevel > 5`